src/room/RoomPage.tsx

```diff
@@ -68,9 +68,13 @@ export const RoomPage: FC = () => {
     // a URL param, automatically register a passwordless user
     if (!loading && !authenticated && displayName && !widget) {
       setIsRegistering(true);
-      registerPasswordlessUser(displayName).finally(() => {
-        setIsRegistering(false);
-      });
+      registerPasswordlessUser(displayName)
+        .finally(() => {
+          setIsRegistering(false);
+        })
+        .catch((e) => {
+          logger.error("Failed to register passwordless user", e);
+        });
```

Comment:
It probably doesn't make a difference here, but for better style, put the `.catch` before the `.finally`.
```suggestion
        .catch((e) => {
          logger.error("Failed to register passwordless user", e);
        })
        .finally(() => {
          setIsRegistering(false);
        });
```